addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.scss

```diff
@@ -6,6 +6,17 @@
     min-width: 0;
 }
 
+.o_ComposerTextInput_mentionDropdown.o-lowPosition {
+    top: 100%;
+}
+
+.o_ComposerTextInput_mentionDropdownPropositionList {
+    &.o-lowPosition {
+        top: 100%;
+        bottom: auto;
+    }
+}
```

Comment:
This doesn't seem to work as expected, when testing mentions in the chatter, the suggestions are covering the whole textarea and we can't see the text anymore.

Reply:
I feel like we should rely on the Popover placement for this, and not try to do CSS ourselves.

Reply:
Illustration of problem it that may help

![image](https://user-images.githubusercontent.com/38213504/83163520-6a65f380-a10b-11ea-8159-cab9428b6b67.png)

Reply:
Any example to do it plz ?

Reply:
I'm not doing anything in particular, I'm just trying to mention someone from a chatter (I'm using contacts app and using the chatter on the partner form since this is one of the fastest modules to install) (and using firefox, but not sure if that makes a difference).